SU-SWS/stanford_profile
# D8CORE-3352: updated configurations for past events

# READY FOR REVIEW

# Summary
- With these config changes, we remove the `stanford_events_past` view, and consolidate it into the `stanford_events` view, similar to what we did with the schedule view.  
- We update the `events/past` page to use the new view display.
- We add/change a few things to match the designs in figma (https://www.figma.com/file/LSfhdmKiOlNniVMvFkvigw/SWS-D8-Events-and-Event-Series-WF-%2B-Mocks?node-id=1823%3A13218)
- Accompanying style changes are in the events module, with this pr: https://github.com/SU-SWS/stanford_events/pull/58

# Needed By (Date)
- 2.0.1 code cutoff

# Urgency
- medium

# Steps to Test

1. checkout this branch.
2. checkout the corresponding branch in the `stanford_events` module (see above)
3. `drush cim`
4. `drush cr`
5. Create some events that occur in the past.  navigate to `events/past` and verify the page looks right, per the Figma mocks (see above)

# Affected Projects or Products
- `stanford_events`: https://github.com/SU-SWS/stanford_events/pull/58

# Associated Issues and/or People
- D8CORE-3352
- See above.
- I'd like to demo this for @ishahism to make sure it meets her approval.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
